usr/src/man/man1m/zpool.1m

```diff
@@ -1342,6 +1382,72 @@ Stop scrubbing.
 .El
 .It Xo
 .Nm
+.Cm trim
+.Op Fl r Ar rate | Fl s
+.Ar pool Ns ...
+.Xc
+Initiates a manual TRIM operation on all of the free space of a pool.
+This informs the underlying storage devices of all of the blocks that
+the pool no longer considers allocated, thus allowing thinly provisioned
+storage devices to reclaim them. Please note that this collects all
+space marked as "freed" in the pool immediately and doesn't wait the
+.Nm zfs_txgs_per_trim
+delay as automatic TRIM does. Hence, this can limit pool corruption recovery
```

Comment:
@BjoKaSH I've already reworded it as follows:

> Please note that this collects all space marked as "freed" in the pool immediately and doesn't wait to age out blocks as automatic TRIM does. Hence, this can limit pool corruption recovery options during and immediately following the manual TRIM to 1-2 TXGs into the past (instead of the standard 32-64 of automatic TRIM).

The change just wasn't committed yet.